hipe4ml/plot_utils.py

```diff
@@ -617,6 +618,33 @@ def plot_feature_imp(df_in, y_truth, model, labels=None, n_sample=10000, approxi
     return res
 
 
+def plot_optuna_results(study):
```

Comment:
I would add the possibility to tune the parameters of the optuna functions you used, they are for the most part common to all functions and 4-5 in total.